lib/host/wasi/inode-win.cpp

```diff
@@ -82,53 +434,310 @@ INode INode::stdErr() noexcept {
   return INode(winapi::GetStdHandle(winapi::STD_ERROR_HANDLE_));
 }
 
-WasiExpect<INode> INode::open(std::string, __wasi_oflags_t, __wasi_fdflags_t,
-                              uint8_t) noexcept {
-  return WasiUnexpect(__WASI_ERRNO_NOSYS);
+WasiExpect<INode> INode::open(std::string Path, __wasi_oflags_t OpenFlags,
+                              __wasi_fdflags_t FdFlags,
+                              uint8_t VFSFlags) noexcept {
+
+  DWORD AttributeFlags = attributeFlags(OpenFlags, FdFlags);
+  DWORD AccessFlags = accessFlags(FdFlags, VFSFlags);
+  // DWORD ShareFlags = shareFlags(OpenFlags);
```

Comment:
Do we still need this?

Reply:
Same as the above comment above.

Reply:
Resolved in [here](https://github.com/WasmEdge/WasmEdge/pull/1826#discussion_r981951772)